src/store/mutations.js

```diff
@@ -179,7 +179,7 @@ export default {
 		envelope.flags[flag] = value
 	},
 	tagEnvelope(state, { envelope, tag, value }) {
```

Comment:
we have to check all usages of `tagEnvelope`. in contrast to `flagEnvelope` the `value` here isn't a boolean but an object, that represents the tag. Alternatively we could also normalize the data like we do with the rest of accounts/mailboxes/messages. In the long run and especially as soon as tags can be renamed and their color changed, we'll want to have the data normalized so the update of one tag is shown everywhere instantly.